src/wp-includes/functions.php

```diff
@@ -9085,3 +9085,41 @@ function wp_is_heic_image_mime_type( $mime_type ) {
 
 	return in_array( $mime_type, $heic_mime_types, true );
 }
+
+/**
+ * Returns a cryptographically secure hash of the message.
+ *
+ * The BLAKE2b algorithm is used by Sodium to hash the message.
+ *
+ * @TODO explain when to use this instead of wp_hash().
+ *
+ * @since x.y.z
+ *
+ * @param string $message The message to hash.
+ * @return string The hash of the message.
+ */
+function wp_hash_value( $message ) {
```

Comment:
An optional parameter to select which salt might be helpful. If nothing else, it can be used for domain separation.